cli/Valet/Configuration.php

```diff
@@ -115,12 +115,21 @@ function writeBaseConfiguration()
         }
 
         /**
-         *  For upgrades, change config key 'domain' to 'tld'
+         * Migrate old configurations from 'domain' to 'tld'
          */
         $config = $this->read();
-        if (isset($config['domain']) && !isset($config['tld'])) {
-            $this->updateKey('tld', $config['domain']);
+
+        if (isset($config['tld'])) {
+            return;
+        }
+
+        $tld = 'test';
+
+        if (isset($config['domain'])) {
+            $tld = $config['domain'];
         }
+
+        $this->updateKey('tld', $tld);
```

Comment:
Could we do:

```php
$this->updateKey('tld', $config['domain'] ?? 'test');
```

Reply:
Valet is designed to support PHP 5.6, so can't use null-coalesce, however, can certainly shorten the code down similarly. 

Pushed squashed update.

Reply:
Whoa I totally thought we had moved past. Good catch!